src/main/java/seedu/duke/TravelActivityList.java

```diff
@@ -211,4 +206,84 @@ public ArrayList<TravelActivity> getTravelActivities () {
         return travelActivities;
     }
 
+    /**
+     * Find all the tasks with a particular tag and prints them out
+     *
+     * @param tag The tag of tasks that the user wants to find
+     */
+
+    public void findTag(String tag){
+        ArrayList<TravelActivity> temporaryArray = new ArrayList<TravelActivity>();;
```

Comment:
It would be better to make use of existing travelActivityList class to shorten code